docs/release-notes/3539-replace-core-experiment-with-new-api.txt

```diff
@@ -0,0 +1,23 @@
+:orphan:
```

Comment:
these look much better than our prior release notes. thanks @gh-determined-ai for the guidance.